docs/api/browser-window.md

```diff
@@ -835,7 +849,7 @@ Moves window to `x` and `y`.
 
 #### `win.getPosition()`
 
-Returns an array that contains window's current position.
+Returns `Integer[]` - Contains the window's current position.
```

Comment:
I don't think this is new syntax, this is pretty standard around the place.  In fact, this is the UML standard for representing collections of a type.